test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetPackCommandTest.cs

```diff
@@ -2182,6 +2182,181 @@ public void PackCommand_ReferencedProjectWithCustomTokensInNuspec()
+            }
+        }
+
+        /// <summary>
+        /// Creates a simple project that defines attributes commonly found in AssemblyInfo.cs
+        /// </summary>
+        /// <remarks>
+        /// The project is created under directory baseDirectory\projectName.
+        /// The project contains just one file called file1.cs.
+        /// </remarks>
+        /// <param name="baseDirectory">The base directory.</param>
+        /// <param name="projectName">The name of the project.</param>
+        /// <param name="referencedProject">The list of projects referenced by this project. Can be null.</param>
+        /// <param name="targetFrameworkVersion">The target framework version of the project.</param>
+        /// <param name="version">The text for the AssemblyVersion attribute.</param>
+        /// <param name="fileVersion">The text for the AssemblyFileVersion attribute.</param>
+        /// <param name="informationalVersion">The text for the AssemblyInformationalVersion attribute.</param>
+        private void CreateTestProjectWithAssemblyInfoAndNuspec(
+            string baseDirectory,
+            string projectName,
+            string version,
+            string fileVersion,
+            string informationalVersion)
+        {
+            var projectContent = @"<Project ToolsVersion='4.0' DefaultTargets='Build'
+    xmlns='http://schemas.microsoft.com/developer/msbuild/2003'>
+  <PropertyGroup>
+    <OutputType>Library</OutputType>
+    <OutputPath>out</OutputPath>
+    <TargetFrameworkVersion>v4.0</TargetFrameworkVersion>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include='file1.cs' />
+  </ItemGroup>
+  <Import Project='$(MSBuildToolsPath)\Microsoft.CSharp.targets' />
+</Project>";
+
+            var projectDirectory = Path.Combine(baseDirectory, projectName);
+            _ = Directory.CreateDirectory(projectDirectory);
+            Util.CreateFile(projectDirectory, projectName + ".csproj", projectContent);
+
+            var csharpContent = @"using System;
+using System.Reflection;
+using System.Runtime.InteropServices;
+
+[assembly: AssemblyTitle(""The Title"")]
+[assembly: AssemblyDescription(""The Description"")]
+[assembly: AssemblyConfiguration("""")]
+[assembly: AssemblyCompany(""The Company"")]
+[assembly: AssemblyProduct(""The Product"")]
+[assembly: AssemblyCopyright(""The Copyright"")]
+[assembly: AssemblyTrademark("""")]
+[assembly: AssemblyCulture("""")]
+
+[assembly: AssemblyVersion(""[[version]]"")]
+[assembly: AssemblyFileVersion(""[[file-version]]"")]
+[assembly: AssemblyInformationalVersion(""[[informational-version]]"")]
+
+namespace " + projectName + @"
+{
+    public class Class1
+    {
+        public int A { get; set; }
+    }
+}";
+
+            Util.CreateFile(projectDirectory, "file1.cs", csharpContent
+                .Replace("[[version]]", version)
+                .Replace("[[file-version]]", fileVersion)
+                .Replace("[[informational-version]]", informationalVersion));
```

Comment:
So we can remove those lines after changing to use variables in the string.

Reply:
Can't remember why I used this weird `[[key]]` convention. I suppose I was inspired by an existing unit test... Replaced with string interpolation.